code/src/components/ThoughtsList.js

```diff
@@ -0,0 +1,36 @@
+/* eslint no-underscore-dangle: 0 */
+
+import React from 'react';
+
+import { formatDistanceToNow } from 'date-fns';
+
+export const ThoughtsList = (props) => {
+  if (props.loading) {
+    return (
+      <h1>Loading...</h1>
```

Comment:
I wouldn't be myself if I didn't come up with some creative tips. This is a great resource for some funny replacement for boring(?) 'loading' message! ;)
https://gist.github.com/meain/6440b706a97d2dd71574769517e7ed32